packages/loader/driver-definitions/src/storage.ts

```diff
@@ -319,4 +319,10 @@ export interface ISummaryContext {
     readonly ackHandle: string | undefined;
 
     readonly referenceSequenceNumber: number;
+
+    /**
+     * Indicates that this is the first summary for a previously created empty file.
+     * The server should use this as the first summary for the file without any corresponding summary op.
+     */
+    readonly initialSummary: boolean;
```

Comment:
Note: I think it's fine for this not to be optional, but only because driver changes propagate slower than runtime (and it's fine to require runtime to provide this flag going forward, but driver may not know about it).
That said, runtime might be talking to old driver who does not understand it, and things will fail. We need to ensure it fails in some reasonable and easily debuggable way, but also that host does its best to avoid such situation.